Src/Sarif.PatternMatcher.Sdk/Fingerprint.cs

```diff
@@ -14,21 +14,14 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk
     public struct Fingerprint
```

Comment:
>Fingerprint [](start = 18, length = 11)

The heart of the change. We delete 'hmac' as unused. We rename 'acct' key to 'account' (as nothing else is abbreviated after this change)'

The following secret keys are all replaced by 'secret' -> key, pat, pwd, skey/128, skey/256, sasToken

Reply:
Is it possible we could have multiple secrets (perhaps different types) in the same match/result?

---
In reply to: [604542444](https://github.com/microsoft/sarif-pattern-matcher/pull/349#discussion_r604542444) [](ancestors = 604542444)

Reply:
Generally, we expect one result per exposed secret. There is a possible problem, though (which may be what you're actually saying), if a single compromised credential has multiple secrets that contribute to it. In this case, currently, you'd need to define a mini-language for the secret value to express all the component secrets. :( If/when this problem emerges, we could take a look at making a change. if you have any immediate thoughts, let me know.

---
In reply to: [605134680](https://github.com/microsoft/sarif-pattern-matcher/pull/349#discussion_r605134680) [](ancestors = 605134680,604542444)

Reply:
Nah, just playing devil's advocate.  Makes sense to me, and I never know where to put things so this simplifies that decision-making significantly.

---
In reply to: [605228417](https://github.com/microsoft/sarif-pattern-matcher/pull/349#discussion_r605228417) [](ancestors = 605228417,605134680,604542444)